ods_ci/tests/Resources/CLI/ModelServing/llm.resource

```diff
@@ -745,21 +746,25 @@ Create PVC And Download Model From S3
     Create File From Template    ${DOWNLOAD_PVC_FILEPATH}    ${DOWNLOAD_PVC_FILLED_FILEPATH}
     ${rc}    ${out}=    Run And Return Rc And Output    oc -n ${namespace} apply -f ${DOWNLOAD_PVC_FILLED_FILEPATH}
     Should Be Equal As Integers    ${rc}    ${0}
-    Wait For Pods To Be Ready    label_selector=name=download-${model_name}    namespace=${namespace}
+    Run Keyword And Continue On Failure   Wait For Pods To Be Ready    label_selector=name=download-${model_name}    namespace=${namespace}
     Wait For Pods To Succeed    label_selector=name=download-${model_name}    namespace=${namespace}
     ...    timeout=${download_timeout}
     ${rc}    ${out}=    Run And Return Rc And Output    oc delete pods -l name=download-${model_name} -n ${namespace}
     Should Be Equal As Integers    ${rc}    ${0}
 
 Extract Service Port
     [Arguments]    ${service_name}    ${namespace}    ${protocol}
-    IF    "${protocol}" == "grpc"
+    ${protocol_name}     Set Variable    name
+    IF    '${protocol}' == 'grpc'
         ${port_name}=    Set Variable    h2c
+    ELSE IF    '${protocol}' == 'TCP'
+            ${protocol_name}=    Set Variable    protocol
```

Comment:
this variable doesn't container the protocol name, though. Maybe you should give it a different name